force-app/main/default/classes/UTIL_BatchJobService.cls

```diff
@@ -154,6 +154,21 @@ public without sharing class UTIL_BatchJobService {
         return getCompletedDaysBetween(batchJob.CompletedDate, batchJob.Status);
     }
 
+    /***
+    * @description Determines number of days passed since the batch job has completed
+    * @param batchId Batch job Id
+    * @return Integer
+    */
+    public Integer getCompleteDaysBetween(Id batchId) {
```

Comment:
I assume the minor difference in spelling is because Id and String don't work as different signatures.  If that's the case, I think we should use a method name that is more different.  This one is confusing since it almost seems like a misspelling.